### DISCUSSION TITLE
Blank icons

### DISCUSSION THREAD

**Discussion by apepindev** [elrumo/macOS_Big_Sur_icons_replacements#1137] [Help with the website] [answered]
Hey, I can't see or download some icons.

<img width="1325" alt="Screenshot 2021-04-06 at 20 33 36" src="https://user-images.githubusercontent.com/2915723/113768103-5e799380-9717-11eb-9217-a43c3befab11.png">

**Comment by cake03** [comment, marked as answer]
the icons have been HACKED so the most downloaded icons have been cached by [Cloudflare](http://www.cloudflare.com) AND ones made after 4 April are visible
https://macosicons.com/blog/hacked

**Reply by apepindev** [reply]
My bad, I missed that.
